### ISSUE TITLE
Global setting

### ISSUE THREAD

**Issue by johanvh83** [Mariusthvdb/custom-ui#33]
Hi Mariusthvdb,

I installed your custom-ui and finally I'm very happy that 1 icon changes color based on a sensor.

So far, I haven't split up my configuration so my configuration.yaml looks like:

![Snap 2021-02-09 at 12 49 11](https://user-images.githubusercontent.com/76775686/107359686-46014a00-6ad5-11eb-9730-1c7df24e8d67.jpg)
(Used a picture, because otherwise it was 1 line of text)

A nice result:
![Snap 2021-02-09 at 12 41 33](https://user-images.githubusercontent.com/76775686/107358893-45b47f00-6ad4-11eb-804f-9e277a3e12ab.jpg)

As you can see I have more temperature sensors and I'd like to use the above for all my temperature sensors.
I checked your page with examples and tried many things. Unfortunately the only thing, I get are errors.

When I change **sensor.living_temp:** into **sensor.*_temp** by using the wildcard I get an error when checking my configuration:
extra keys not allowed @ data['customize']['sensor.*_temp']

Spend already some hours to solve this but can't find it. Can you put me in the right direction? Many thanks!

**Comment by Mariusthvdb** [comment]
did you write that under the key
```
homeassistant:
  customize_glob:
```

**Comment by johanvh83** [comment]
No, I made the code as above.

Changed code to:

`homeassistant:
   customize_glob:
    sensor.*_temp:
      templates:
        icon_color: >
          if (state < 5) return 'rgb(0,0,255)';
          if (state < 20) return 'rgb(25,0,255)';
          if (state < 20,5) return 'rgb(50,0,200)';
          if (state < 21) return 'rgb(70,0,180)';
          if (state < 21,5) return 'rgb(90,0,168)';
          if (state < 22) return 'rgb(112,0,144)';
          if (state < 22,5) return 'rgb(134,0,121)';
          if (state < 23) return 'rgb(156,0,99)';
          if (state < 23,5) return 'rgb(189,0,66)';
          if (state < 24) return 'rgb(211,0,44)';
          if (state < 24,5) return 'rgb(233,0,22)';
          if (state < 25) return 'rgb(255,0,0)';
          return 'rgb(0,0,0)';`

and it works for all temperature sensors. Many thanks!

**Comment by Mariusthvdb** [comment]
great! closing ;-)

**Comment by johanvh83** [comment]
Hmmm,
Changed from configuration.yaml to customize_glob.yaml and included this file in the configuration. Also changed some rgb colors and temperature levels as per below:

`sensor.*_temp:
      templates:
        icon_color: >
          if (state < 19) return 'rgb(0,0,255)';
          if (state < 20) return 'rgb(25,0,255)';
          if (state < 21) return 'rgb(50,0,200)';
          if (state < 22) return 'rgb(70,0,180)';
          if (state < 23) return 'rgb(90,0,168)';
          if (state < 24) return 'rgb(112,0,144)';
          if (state < 25) return 'rgb(134,0,121)';
          if (state < 26) return 'rgb(156,0,99)';
          if (state < 27) return 'rgb(189,0,66)';
          if (state < 28) return 'rgb(211,0,44)';
          if (state < 29) return 'rgb(233,0,22)';
          if (state < 30) return 'rgb(255,0,0)';
          return 'rgb(0,0,0)';`

Kitchen temp is 21,6 this should be rgb(50,0,200) and this is correct:
![Schermafbeelding 2021-02-09 om 15 36 31](https://user-images.githubusercontent.com/76775686/107379144-ee6ed880-6aec-11eb-9fa8-a7be205b9126.png)

Living temp is 24,7, so this must be rgb(112,0,144) but it is also shown as rgb(50,0,200), same as kitchen:
![Schermafbeelding 2021-02-09 om 15 37 50](https://user-images.githubusercontent.com/76775686/107379166-f29af600-6aec-11eb-8ea4-afc1ec24bc0a.png)

Any idea why?

**Comment by Mariusthvdb** [comment]
what are the sensor entity_id's?

jouwe got to be precise, and your glob is customizing sensor ending their entity_if with '_temp'.
If these entity_id's end with their respective locations ('living', 'keuken'), the glob won't work, and you have to change the glob accordingly

**Comment by johanvh83** [comment]
Just restarted HA and it worked, without any changes.
Issue may be closed. THANKS!

**Comment by johanvh83** [comment]
Sorry to disturb again. May I ask you 1 more question?
For the temperature sensors I had already created the sensors based on the state attributes 'temperature'.

Now I'd like to copy/paste above method for my lights.
I want to change the bulb icon, based on the intensity of the bulb, for all my light entities.
Does this mean I have to create first sensors for all my individual light entities?

I made this custumize_glob.yaml:
![Schermafbeelding 2021-02-09 om 16 50 03](https://user-images.githubusercontent.com/76775686/107389193-def48d00-6af6-11eb-91fc-4043e72c991c.png)

But the icons return to rgb(0,0,0) because it's not linked to the brightness
